### ISSUE TITLE
Brand request: oracle-object-storage

### ISSUE THREAD

**Issue by msays2000** [FortAwesome/Font-Awesome#19202]
### Use case

[Rclone documentation](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md) needs an icon for storage providers, oracle object storage is a new backend storage addition

### Example image

![image](https://user-images.githubusercontent.com/7993290/180428945-473a96a6-c075-42a7-8cc7-b693122e7705.png)

![image](https://user-images.githubusercontent.com/7993290/180429576-8c595353-129a-4010-a757-558acbca00b1.png)

### Icon request checklist

- [X] The title starts with "Icon request: " and is followed by the requested icon name
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/new#faqs)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Could you please clarify is this is a brand, and, if it is not, how Oracle is related to this icon?

**Comment by msays2000** [comment]
yes this is a brand, oracle object storage is provided by oracle cloud infrastructure :
https://docs.oracle.com/en-us/iaas/Content/Object/Concepts/objectstorageoverview.htm 

if you are interested in including Oracle icons. Please consider to upload them from here:
https://docs.oracle.com/en-us/iaas/Content/General/Reference/graphicsfordiagrams.htm 
https://docs.oracle.com/en-us/iaas/Content/Resources/Assets/OCI_Icons_PNG_SVG.zip

**Comment by tagliala** [comment]
> yes this is a brand

I will change this request to a brand request

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No worries.
* This brand has gone the way of the dodo.
